pkg/agent/route/route_linux.go

```diff
@@ -67,6 +78,9 @@ var (
 	// IPTablesSyncInterval is exported so that sync interval can be configured for running integration test with
 	// smaller values. It is meant to be used internally by Run.
 	IPTablesSyncInterval = 60 * time.Second
+
+	serviceGWHairpinIPv4 = net.ParseIP("169.254.169.253")
```

Comment:
We definitely should define the constants multiple times. Please define them in a single file, and import from others.

Reply:
How about pkg/agent/config? If in the package, we should add a new file which is only to contain this two variables.

Reply:
pkg/agent/config sounds good to me. Maybe adding to node_config.go is fine too, if we want not to add a new file.